moderne-platform-adventure/README.md

```diff
@@ -1,98 +1,113 @@
-# Moderne Platform Adventure
+# Moderne platform Adventure
 
-In this adventure, there are two challeges:
+In this adventure, you will utilize the [Moderne platform](https://public.moderne.io/) to:
 
-- The first one, which is all about migrating the spring-petclinic project from a UI 
-like we have completed in the other exercises. 
+* Migrate the [Spring
+  PetClinic](https://github.com/spring-projects/spring-petclinic) repository
+  from Spring Boot 2 to 3
+* Fix security vulnerabilities across hundreds of open-source projects
 
-- The second one is about fixing all the security vulnerabilities across hundreds of projects.
+## Prepare your environment
 
+Go to [https://public.moderne.io/](https://public.moderne.io/) and register with
+your GitHub account. Once you've signed in, you'll find more than 30,000
+open-source repositories that can be used to test OpenRewrite recipes without
+you having to configure anything.
 
-## Prepare environment
+*Note: If you're interested in adding your own repository to the Moderne
+platform, please see the [CLI adventure](/moderne-cli-adventure/README.md)*
 
-Visit https://public.moderne.io/ and register yourself with your GitHub account. Moderne has more than 30k open source repositories loaded that can be used to test the OpenRewrite recipes without having to configure 
-anything in your build system. If you are interested to see how you can publish a new open source repository, we recommend to also play
-with  the Moderne CLI adventure.
+## Migrating to Spring Boot 3 with the Moderne platform
 
-## Spring Boot Migration using the Moderne Platform
-
-1. Now, we are going to create a repository group with one repository that contains the
-Spring Petclinic content when it was using a 2.x Spring Boot version. To do so,
-click on the `Create` option of the `Repository Groups` section.
+1. Once you're logged in to [Moderne](https://public.moderne.io/), the first
+   thing you'll want to do is to create a repository group with just the Spring
+   PetClinic repository in it. To do so, click on the `Default` group in the top
+   right-hand corner of your screen and select `Create` from the menu that
+   appears:
 
 ![context menu](assets/menu.png)
 
-2. A repository group is a list of repositories that are grouped under a name that it is
-only visible for your user. You can use any name for the repository group. We are going
-to call it `petclinic`. After giving a name, you need to select the repository 
-`github.com/timtebeek/spring-petclinic@2.0.0`. Then click on the `Save` option.   
+2. A repository group is a list of repositories that are grouped under a name
+that is only visible to you. You can use any name for the repository group. For
+this adventure, let's call it `petclinic`. After giving it a name, select
+`github.com/timtebeek/spring-petclinic@2.0.0` from the list of available
```

Comment:
I could not find this repository in Moderne. I left it in for now, but we should either update these instructions or add this repository to Moderne.

Reply:
I can see it.. so I think that it is a matter of the instructions but I will also test it with Joan

Reply:
I have found the issue, only moderne team members can see all the repositories. Working on making the repository visible under the Spring organization.